chrome/content/zotero/xpcom/itemTreeView.js

```diff
@@ -260,6 +261,44 @@ Zotero.ItemTreeView.prototype.setTree = Zotero.Promise.coroutine(function* (tree
 });
 
 
+Zotero.ItemTreeView.prototype.setSortColumn = function() {
+	var colId, col, dir, currentCol, currentDir;
```

Comment:
This discussion will come up next time someone else starts working with this codebase (and even Aurimas comments on it, because the rule "var for function scope, let for block scope" is kind of ambiguous).
I am fine with sticking to the rule, but we should at least be more lenient about PR comments for variable declarations where both could reasonably make sense.